locales/tl-PH.json

```
@@ -0,0 +1,630 @@
+    "blocked_by": "Ikaw ay na-block ng user na ito.",
+    "blocked_domains": "Mga na-block na domains",
+    "blocked_users": "Mga na-block na users",
+    "blocking": "Naka-block",
+    "bot": "BOT",
+    "favourites": "Mga paborito",
+    "follow": "Sundan",
+    "follow_back": "Sundan balik",
+    "follow_requested": "Humiling ng pagsunod",
+    "followers": "Mga tagasunod",
+    "followers_count": "{0} Mga tagasunod|{0} Tagasunod|{0} Mga tagasunod",
+    "following": "Mga sinusundan",
+    "following_count": "{0} Sinusundan",
+    "follows_you": "Nagsusunod sa'yo",
+    "go_to_profile": "Pumunta sa profile",
+    "joined": "Sumali",
+    "moved_title": "nagpahiwatig na ang kanilang bagong account ay:",
+    "muted_users": "Mga na-mute na users",
+    "muting": "Naka-mute",
+    "mutuals": "Mga mutual",
+    "notifications_on_post_disable": "Huwag nang magbigay ng abiso kapag nagpopost si {username}",
+    "notifications_on_post_enable": "Magbigay ng abiso kapag nagpopost si {username}",
+    "pinned": "Nakapin",
+    "posts": "Mga post",
+    "posts_count": "{0} Mga post|{0} Post|{0} Mga post",
+    "profile_description": "Header ng profile ni {0}",
+    "profile_personal_note": "Personal na Tala",
+    "profile_unavailable": "Hindi magamit ang profile",
+    "request_follow": "Humiling ng pagsunod",
+    "unblock": "Tanggalin ang block",
+    "unfollow": "Huwag sundan",
+    "unmute": "Tanggalin ang mute",
+    "view_other_followers": "Ang mga tagasunod mula sa ibang mga instance ay hindi maipapakita.",
+    "view_other_following": "Ang mga sinusunod mula sa ibang mga instance ay hindi maipapakita."
+  },
+  "action": {
+    "apply": "Mag-apply",
+    "bookmark": "I-bookmark",
+    "bookmarked": "Nai-bookmark na",
+    "boost": "Mag-boost",
+    "boost_count": "{0} ang bumuo",
+    "boosted": "Na-boost na",
+    "clear_publish_failed": "Tanggalin ang mga error sa pag-publish",
+    "clear_upload_failed": "Tanggalin ang mga error sa pag-upload ng file",
+    "close": "Isara",
+    "compose": "Mag-compose",
+    "confirm": "Kumpirmahin",
+    "edit": "I-edit",
+    "enter_app": "Pumasok sa App",
+    "favourite": "I-favorite",
+    "favourite_count": "{0} ang paborito",
+    "favourited": "Na-favorite na",
+    "more": "Iba pa",
+    "next": "Susunod",
+    "prev": "Nakaraan",
+    "publish": "I-publish",
+    "reply": "Sumagot",
+    "reply_count": "{0} ang sumagot",
+    "reset": "I-reset",
+    "save": "I-save",
+    "save_changes": "I-save ang mga pagbabago",
+    "sign_in": "Mag-sign in",
+    "sign_in_to": "Mag-sign in sa {0}",
+    "switch_account": "Magpalit ng account",
+    "vote": "Bumoto"
+  },
+  "app_desc_short": "Isang mabilis na web client ng Mastodon",
+  "app_logo": "Logo ng Elk",
+  "app_name": "Elk",
+  "attachment": {
+    "edit_title": "Paglalarawan",
+    "remove_label": "Tanggalin ang attachment"
+  },
+  "command": {
+    "activate": "I-activate",
+    "complete": "Tapos na",
```

```suggestion
    "complete": "Tapusin",
```

I think this should be an action